script/Arcueid/Day 2/02_00_ARC02_1C.txt

```diff
@@ -231,7 +231,7 @@ It's 7:30 in the morning.
 [sha:fe8502ef15211c08c39a1dc2bbd764c1234a20d1]{
 -- Page 107, Offset 2882.
 -- 道を小走りで歩く学生服姿は自分しか見あたらなかった。
-I can't spot anyone else wearing a school uniform as I jog down the street.
+As I jog down the street, I fail to spot anyone else dressed in a school uniform.
```

Comment:
`fail to spot` maybe has a subtle vibe of him _searching_ for people in uniforms, where I think this is just more of a remark.

Reply:
"As I jog down the street, I don't see anyone else wearing a school uniform."